script/dashboard/dashboard.js

```diff
@@ -157,3 +155,24 @@ app.controller('postcodeMapCtrl', function($scope, $http){
         });
     });
 });
+
+app.service('colorService', function() {
+    var o = {
+
+    };
+
+    o.getColor = function(ratio) {

```

Comment:
Jak damy var to nie bedzie mozna jej użyć z zewnątrz, a jeśli się myle to pokaż jak, bo ten sposób co podałes nie działa :)